app/models/data.py

```diff
@@ -37,6 +37,9 @@ def to_dict(self):
 
         return d
 
+    def get(self, key):
```

Comment:
would `getattr` work instead? I've used that in some other places in the code

Reply:
From what I can tell, this is only used in the `csvcommon.py` part where you have `datum.get(column.model_column)`? I just poked at this a bit and I think the issue was that sometimes `datum` is a dict (so `getattr` would fail but `.get` works) and sometimes it's a SQLAlchemy object. The correct thing would probably be to never have it be a dict; the failure I ran into was at `tests/app/public_test.py::test_get_states_daily`, getting `*** AttributeError: 'dict' object has no attribute 'date'`.